src/seam-connect/client.ts

```diff
@@ -111,4 +151,51 @@ export class Seam extends Routes {
       throw error
     }
   }
+
+  static async getClientSessionToken(
+    ops: CSTParams
+  ): Promise<APIResponse<ClientSessionResponse>> {
+    const { apiKey, endpoint, axiosOptions } =
+      getSeamClientOptionsWithDefaults(ops)
+    let headers: AxiosRequestHeaders = {
+      ...axiosOptions?.headers,
+    }
+
+    if (ops.publishableKey?.startsWith("seam_pk")) {
+      // frontend mode
+      headers["seam-publishable-key"] = ops.publishableKey
+    } else if (apiKey?.startsWith("seam_")) {
+      // backend mode
+      headers["seam-api-key"] = apiKey
+    }
+    if (ops.userIdentifierKey) {
+      headers["seam-user-identifier-key"] = ops.userIdentifierKey
+    } else {
+      throw new Error("userIdentifierKey is required")
+    }
+    try {
+      const response: AxiosResponse & {
```

Comment:
@azat-co I didn't save my comment 😭 

Anyway, was just going to say we should make sure to respect `axiosOptions`, so like in the constructor, something like

```ts
    const client = axios.create({
      ...axiosOptions,
      baseURL: endpoint,
      headers,
    })

   await client.post(...)
```


Looking closer, I just realized the normal client behavior does a bit more, like always throw `SeamAPIError`. We should factor out the client creation method and use it both places. I will make an issue for that.

Reply:
https://github.com/seamapi/javascript/issues/184

Reply:
Yes, basically we should follow the same logic as the constructor for how we use `axios`. I created a [new issue that better explains the desired behavior](https://github.com/seamapi/javascript/issues/184). It's ok if we resolve that issue in a follow-up PR.